usagov/vote-gov-drupal
# VOTE-981 add views_data export module

Add ability to export NVRF reports as csv



## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-981

## Description

Add ability to download NVRF data reports as csvs.

## Deployment and testing

### Post-deploy

1. run `lando retune`

### QA/Test

1. visit http://vote-gov.lndo.site/admin/nvrf-fields/export and make sure that it exports a csv of data
2. visit http://vote-gov.lndo.site/admin/nvrf-state-report/export and make sure that it exports a csv of data
3. visit http://vote-gov.lndo.site/admin/nvrf-state-report/fields/export and make sure that it exports a csv of data

## Checklist for the Developer




## Checklist for the Peer Reviewers
